shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Fees.hs

```diff
@@ -499,3 +519,6 @@ sizeTests =
   where
     p :: Proxy C
     p = Proxy
+
+runLocalTest :: IO ()
```

Comment:
not sure if you meant to leave this in

Reply:
It has been removed.

Reply:
it looks like it is still there, did you mean to push another commit? thanks!